docs/docs/developers/contracts/writing_contracts/functions/write_constructor.md

```diff
@@ -2,36 +2,32 @@
 title: How to write a constructor
 ---
 
-This page explains how to write a constructor function.
+This page explains how to write a constructor (or initializer) function.
 
-To learn more about constructors, read [this](./main.md#constructors).
+To learn more about constructors/initializers, read [this](./main.md#initializer-functions).
```

Comment:
Feels like we should settle for a name. I'm fine with either.

Reply:
I like initializer because the behavior is bit different than constructors in other contexts. will update